File: benchmarks/poly/fibonacci/fibonacci.js
function fib(n) {
  if (n < 2) return n;
  return fib(n - 1) + fib(n - 2);
}

function main() {
  const _t0 = process.hrtime.bigint();
  for (let i = 0; i < 10; i++) {
    console.log(fib(i));
  }
  console.error(`bench_ns=${process.hrtime.bigint() - _t0}`);
}

main();

File: benchmarks/poly/fibonacci/fibonacci.rs
fn fib(n: i64) -> i64 {
    if n < 2 {
        return n;
    }
    fib(n - 1) + fib(n - 2)
}

fn main() {
    let t0 = std::time::Instant::now();
    for i in 0..10 {
        println!("{}", fib(i));
    }
    eprintln!("bench_ns={}", t0.elapsed().as_nanos());
}
